operatorhub/openshift/config.yaml

```diff
@@ -236,6 +236,15 @@ image-substitutions:
         envKeys:
           - IMAGE_ADDONS_PARAM_MAVEN_IMAGE
 
+# pipelines console plugin image
+- image: ghcr.io/openshift-pipelines/console-plugin:main
```

Comment:
lets create an issue, this image needs to be renamed with redhat offical image url